apps/test-netzer-encoder/src/Netzer.cpp

```diff
@@ -0,0 +1,57 @@
+#include "Netzer.h"
+
+using namespace Encoder;
+Netzer::Netzer(PinName mosi, PinName miso, PinName sclk, 
+									callback_ptr callback) 
+									: m_spi(mosi, miso, sclk), m_callback(callback){
+	m_spi.format(16,3);
```

Comment:
Alright cool, I double checked the documentation and I believe it is mode 3 (inverted phase, rising edge)

Reply:
I think it should be mode 2 actually, from the timing diagram it looks like data should be sampled on the falling edge of the clock. It's the same as the aeat-6012, but this might mean the data actually starts on the 2nd sampled bit